documentation/docs/components/video.md

```diff
@@ -0,0 +1,18 @@
+# Video
+
+## Example
+
+| type     | name       |parameter_list |
+|--------- | -----------| ------        |
+|video     |video_src	|subtitles_src: plh_video/lets_slow_down.vtt |
```

Comment:
This component also uses the value column, right? The `.mp4` file needs to be read from somewhere!